ci-operator/templates/openshift/installer/cluster-launch-installer-libvirt-e2e.yaml

```diff
@@ -193,8 +193,14 @@ objects:
         export PATH=/home/packer:$PATH
         function run-tests() {
           mkdir -p /tmp/artifacts/junit
-          if which openshift-tests && [[ -n "${TEST_SUITE-}" ]]; then
-            openshift-tests run "${TEST_SUITE}" \
+          if which openshift-tests; then
+            if [ "${TEST_SUITE}" != "run-with-file" ]; then
+                openshift-tests run "${TEST_SUITE}" \
+                -o /tmp/artifacts/e2e.log --junit-dir /tmp/artifacts/junit
+            exit 0
+            fi
+            curl -L -O https://raw.githubusercontent.com/code-ready/crc/master/test/extended/openshift-conformance-test
```

Comment:
I don't like this floating in an external location.  Can you inline it?  Maybe in your job definition, and pass it in as a parameter to this template?

Reply:
> I don't like this floating in an external location. Can you inline it? Maybe in your job definition, and pass it in as a parameter to this template?

@wking I do agree but this is a temp for us and might be remove once we have one of the suite completely working for libvirt side also https://raw.githubusercontent.com/code-ready/crc/master/test/extended/openshift-conformance-test allow us adding the test cases disjoint to this repo for time being and putting PR when add a single test to this file.